### PR TITLE
Feature/filter updates

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

File: src/js/views/eva-variant-view.js [hunk 1]
```diff
@@ -214,14 +214,11 @@ EvaVariantView.prototype = {
 
     },
     _renderConsequenceTypeData: function (data) {
-
-        var annotation = data[0].annotation.consequenceTypes;
+        var _this = this;
+        var annotation = data[0].annotation.consequenceTypes.sort(_this._sortBy('ensemblGeneId', _this._sortBy('ensemblTranscriptId')));
```
Comment (cyenyxe) [inline src/js/views/eva-variant-view.js]: Would using "this" directly cause any issues?

Comment (kandaj) [inline src/js/views/eva-variant-view.js]: no. i'm referencing into temporary variable.

Comment (cyenyxe) [inline src/js/views/eva-variant-view.js]: If using "this" will work well, the extra assignment doesn't add anything useful and could be removed.

**Comment by cyenyxe** [comment]
f632f50 states to have "variant view transcripts sorted" but most of what it does is removing a lot of bower_component files, making very complicated to review the rest of the code. Could that commit be taken out to another branch and PR?

**Pull request merged**